src/eureka/S5_lightcurve_fitting/models/GPModel.py

```diff
@@ -367,41 +288,109 @@ def get_kernel(self, kernel_name, i, channel=0):
         AssertionError
             Celerite currently only supports a Matern32 kernel.
         """
-        # get metric for the individual kernel for the current channel
-        metric = (1./np.exp(self.coeffs[kernel_name][channel]))**2
-
         if self.gp_code_name == 'george':
+            # get metric and amplitude for the current kernel and channel
+            amp = np.exp(self.coeffs[c, k, 0]*2)  # Want exp(sigma)^2
```

Comment:
Do we want an amplitude for each individual kernel or one amplitude for the sum of all kernels? I'd usually do the latter.

Reply:
I'd think an amplitude for each kernel would make sense. For example, if you had a large amplitude periodic component and a small amplitude more-nuanced component, I'd want to reflect that in the coefficients for each kernel. But I'm not the most advanced GP user and I could definitely be wrong about what the best choice is.

Reply:
Yes, you're right. With 'time' as the only kernel input at the minute I assume it makes the most sense to do it this way.